src/redux/auth/auth.sagas.js

```diff
@@ -36,7 +36,7 @@ function* handleAdminLoad({ payload }) {
 
 function* handleAdminCheckByToken() {
   try {
-    const authToken = getFromLocalStorage('HORONDI_AUTH_TOKEN');
+    const authToken = yield call(getFromLocalStorage,'HORONDI_AUTH_TOKEN');
```

Comment:
There is no need to use `yield call` for access to the local storage because it is sync action from the js point of view

Reply:
fixed